health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/hooks/services/searchLocalisationService.js

```
@@ -36,7 +36,15 @@ const searchLocalisationService = async ({ tenantId, module, locale, params = {}
         params: { ...params, tenantId: tenantId, module: module, locale: locale },
         body: {},
       });
-      return response?.messages; // Indicate success
+
+      const result = [];
+      response?.messages?.forEach(({ code, message, module }) => {
+        let item = { code, module };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'item' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)